### PR TITLE
tripform tests

### PR DESCRIPTION
**Pull request by Quetourah**
checking if the tests pass/work

### CODE REVIEW

**Comment by Quetourah** [comment]
awesome. thanks!

File: src/components/TripForm/TripForm.test.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+describe('TripForm', () => {
+	it('should match the snapshot', () => {
+		const wrapper = shallow(<TripForm />);
+		expect(wrapper).toMatchSnapshot();
+	});
+	it('changes page to Trip Guests', () => {
+		const wrapper = shallow(<TripForm />);
+		wrapper.setState({ page: 2 });
+		expect(wrapper.state('page')).toEqual(2);
+	});
```
Comment (kcode20) [inline src/components/TripForm/TripForm.test.jsx]: This test seems a little bit redundant. You are setting the `this.state.page` to equal `2`, then testing if the value is `2`. This test would still be true even if the page did not change the page to `<TripGuests/>`.

File: src/components/TripForm/TripForm.test.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	it('changes page to Trip Rest Stops', () => {
+		const wrapper = shallow(<TripForm />);
+		wrapper.setState({ page: 3 });
+		expect(wrapper.state('page')).toEqual(3);
+	});
+});
```
Comment (kcode20) [inline src/components/TripForm/TripForm.test.jsx]: I would also like to see tests on the functions on the components class. We should be testing that `nextPage` and `previousPage` works.

Comment (jhrbva) [reply src/components/TripForm/TripForm.test.jsx]: Can we pair on this one?

**Pull request merged**